CITATION.cff

```diff
@@ -0,0 +1,23 @@
+# Parser settings.
+cff-version: 1.2.0
+message: Please cite this crate using these information.
+
+# Version information.
+date-released: 2022-11-12
+version: 0.4.23
+
+# Project information.
+abstract: Date and time library for Rust
+authors:
```

Comment:
This list is mandatory.  It specifies who has authored the project.  An author can be either a natural person, a company or an anonymous group of people.  I failed to deduce the authors of `chrono` from the organisation, the repository, the Rust manifest and the license.  I would like to discuss who should be named here.

Reply:
I think it would do for now to list @lifthrasiir, @esheppa, @quodlibetor and me.

Reply:
Alright, I will add these information.

Reply:
I added the author information based on the public data from their GitHub accounts.  Please tell me if further adjustments are required, especially if I should have entered name parts in the wrong field(s).